src/sctools/groups.py

```diff
@@ -0,0 +1,127 @@
+"""
+Group QC outputs
+
+"""
+
+from crimson import picard
+import argparse
+import os
+import pandas as pd
+
+
+def aggregate_picard_metrics_by_row(file_names, output_name):
+    """
+    pipeline output picard QC metrics at sinle cell/sample level.
+    This functin is called to merge/aggregate QC metrics by metrics type and
+    then merge multiple QC measurement into single matrix file.
+    In this file, column is sample/cell and row is QC metrics
+    :param files: metric files from pipeline outputs
+    :param met_name: metrics name with workflow name and subworkflow name
```

Comment:
Can you please change these docstring parameters to `file_names` and `output` name to reflect the current function parameters?